virttest/utils_test/qemu.py

```diff
@@ -1571,53 +1657,6 @@ def unplug_memory(self, vm, name):
         return devices
 
     @error_context.context_aware
-    def get_guest_mem(self, vm):
-        """
-        Get physical memory size detect by guest os
-        """
-        error_context.context("Get physical memory in guest")
-        mem_size = 0.0
-        session = self.get_session(vm)
-        if self.params.get("os_type") == "windows":
-            cmd = 'systeminfo | findstr /C:"Total Physical Memory"'
-            regex = re.compile("Total Physical Memory:\s+(.*)", re.M | re.I)
-        else:
-            cmd = 'cat /proc/meminfo'
-            regex = re.compile("MemTotal:\s+(.*)", re.M | re.I)
-        output = session.cmd_output_safe(cmd, timeout=240)
-        try:
-            mem_str = regex.search(output.replace(',', '')).groups()[0]
-            mem_size = self.normalize_mem_mb(mem_str)
-            logging.info(
-                "Memory reported by OS: %.2f %sB" %
-                (mem_size, self.UNIT))
-        except Exception:
-            logging.warn("Invalid outputi from guest: %s" % output)
-        return mem_size
-
-    @error_context.context_aware
-    def get_vm_mem(self, vm):
-        """
-        Get guest memory
-        """
-        error_context.context("Get memory assign to VM")
-        mem_size = 0.0
-        for device in vm.devices:
-            if isinstance(device, qdevices.Dimm):
-                mem_qid = device.get_param("memdev")
-                mem = vm.devices.get_by_qid(mem_qid)[0]
-                mem_size += self.normalize_mem_mb(mem.get_param("size"))
-        if self.params.get("mem"):
-            mem_str = self.params.get("mem")
-            if mem_str.isdigit():
-                mem_str = "%s MB" % mem_str
-            mem_size += self.normalize_mem_mb(mem_str)
-        logging.info(
-            "Memory assigned to VM: %.2f %sB" %
-            (mem_size, self.UNIT))
-        return mem_size
-
-    @error_context.context_aware
```

Comment:
This line is not dropped.

Checked the code:
    @error_context.context_aware
    def get_mem_addr(self, vm, qid):
        """
        Get guest memory address from qemu monitor.
        """